.github/workflows/unit_test_cpu.yaml

```diff
@@ -23,4 +23,6 @@ jobs:
         conda activate "${CONDA_ENV}"
 
         pip install --pre torch --index-url https://download.pytorch.org/whl/nightly/cu121
+        pip install --pre torchdata --index-url https://download.pytorch.org/whl/nightly
+        pip install -e ".[dev]"
```

Comment:
ditto

Reply:
The pip install? I wasn't able to run the unit test without it, it says torchtitan package is not found

Reply:
I just tried: it will work if you have `__init__.py` in the `test/datasets` subfolder. Alternatively you can just put the .py test file in `test/`